build.sbt

```diff
@@ -547,6 +547,19 @@ lazy val mainSettingsProj = (project in file("main-settings"))
       // added a method to a sealed trait
       exclude[InheritedNewAbstractMethodProblem]("sbt.Scoped.canEqual"),
       exclude[InheritedNewAbstractMethodProblem]("sbt.ScopedTaskable.canEqual"),
+
+      // redefined implicits using SAM type shorthand, without Append.Sequence
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendSeqImplicit"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendSource"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendList"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendVectorImplicit"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendClasspath"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendListImplicit"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendOption"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendMap"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendSeq"),
+      exclude[DirectMissingMethodProblem]("sbt.Append.appendSet"),
+      exclude[MissingClassProblem]("sbt.Append$Sequence"),
```

Comment:
Are these exclusions actually safe? Is it possible that plugins use them (implicitly)?

Reply:
I think it would be safer to leave them defined but deprecate them and remove the implicit modifier.

Reply:
Good idea.